tensorflow/lite/kernels/concatenation_test.cc

```diff
@@ -26,9 +26,24 @@ using ::testing::ElementsAreArray;
 
 class BaseConcatenationOpModel : public SingleOpModel {
  public:
-  // TODO(ahentz): Also test different activation types, axis, input
-  // dimensions.
   BaseConcatenationOpModel() {}
+  BaseConcatenationOpModel(
```

Comment:
It seems the new "BaseConcatenationOpModel" class is very similar to existing one with the same name. Could you please merge the two?

Something like this:
BaseConcatenationOpModel(const TensorData& input_template, int axis,
                           int num_inputs, const ActivationFunctionType& fused_activation_function = ActivationFunctionType_NONE) {
.....
}

Reply:
@kyscg gentle ping